tests/src/requestExit.v2.test.js

```diff
@@ -0,0 +1,51 @@
+import 'core-js/stable';
+import 'regenerator-runtime/runtime';
+import { waitForAppScreen, zemu, genericTx, nano_models } from './test.fixture';
+import { ethers } from 'ethers';
+import { parseEther } from 'ethers/lib/utils';
+import { ledgerService } from '@ledgerhq/hw-app-eth';
+
+const contractAddr = '0x380c7e0ec45f5b62ae5a0a6d6a0b95b532e3dd9c';
+
+const pluginName = 'Kiln';
+const abi_path = `../cal/abis/${contractAddr}.json`;
+const abi = require(abi_path);
+
+nano_models.forEach(function (model) {
+  test(
+    '[Nano ' + model.letter + '] RequestExit V2 Eth',
+    zemu(model, async (sim, eth) => {
+      const contract = new ethers.Contract(contractAddr, abi);
+
+      const { data } = await contract.populateTransaction.requestExit(42);
```

Comment:
42 wei is very small for the test no ?

Reply:
we don't clear sign it anyway as it's shares amount